techMail.js

```diff
@@ -34,11 +34,11 @@ module.exports = function(configuration, templatesDir) {
 
             emailTemplates(templatesDir, function(err, template) {
                 if (err) {
-                    logger.error(err);
+                    logger.error("Unable to configure the template engine... - Details:", err && err.stack || err);
                 } else {
                     template(options.template, locals, function(err, html, text) {
                         if (err) {
-                            logger.error(err);
+                            logger.error("Unable to render the template '" + options.template + "' - Details:", err && err.stack || err);
```

Comment:
It should be the same using either '+' or ',' to build the error message, in the case `options.template` is null or undefined or an empty String